addons/point_of_sale/static/src/app/screens/partner_list/partner_list.xml

```diff
@@ -1,80 +1,50 @@
-                    <i t-if="ui.isSmall" class="fa fa-angle-double-left"/>
-                    <t t-else=""> Discard</t>
-                </div>
-                <div class="top-right-buttons d-flex gap-2 w-100 justify-content-end">
-                    <div t-if="state.detailIsShown and state.editModeProps.partner.id" class="button more-info">
-                        <a t-on-click="goToOrders" class="btn btn-lg btn-outline-secondary" target="_blank"> Orders </a>
-                    </div>
-                    <div t-if="state.detailIsShown &amp;&amp; state.editModeProps.partner.id" class="button more-info">
-                        <a t-att-href="partnerLink" class="btn btn-lg btn-outline-secondary" target="_blank"> More info</a>
-                    </div>
-                    <div class="search-bar-container sb-partner d-flex align-items-center w-100 w-md-70 w-lg-50 h-100 h-100" t-if="!state.detailIsShown">
-                        <div class="input-group h-100">
-                            <div class="pos-search-bar form-control d-flex align-items-center bg-view">
-                                <i class="oi oi-search me-2" t-on-click="_onPressEnterKey"/>
-                                <input class="flex-grow-1 w-auto h-100 border-0 p-2 me-4" type="text" autofocus="autofocus" t-ref="search-word-input-partner" placeholder="Search Customers..." size="1" t-on-keyup="updatePartnerList" />
-                                <i class="fa fa-times position-absolute end-0 me-2 pe-1 cursor-pointer" t-on-click="_clearSearch"/>
-                            </div>
-                        </div>
-                    </div>
-                </div>
+                <Input tModel="[state, 'query']"
+                    class="'ms-auto'"
+                    isSmall="ui.isSmall"
+                    placeholder="'Search Customers...'"
+                    icon="{type: 'fa', value: 'fa-search'}"
+                    autofocus="true"
+                    debounceMillis="100" />
+            </t>
+            <table class="table">
+                <thead t-if="!ui.isSmall">
+                    <tr>
+                        <th class="py-2">Name</th>
+                        <th class="py-2">Address</th>
+                        <th class="partner-line-email py-2">Contact</th>
+                        <th class="pos-right-align py-2" t-if="isBalanceDisplayed">Balance</th>
+                        <th class="py-2"></th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <PartnerLine t-foreach="partners" t-as="partner" t-key="partner.id"
+                                partner="partner"
+                                isSelected="props.partner?.id === partner.id"
+                                isBalanceDisplayed="isBalanceDisplayed"
+                                onClickEdit.bind="editPartner"
+                                onClickUnselect.bind="() => this.clickPartner()"
```

Comment:
indeed

Reply:
this change caused a stupid problem...
`onClickUnselect` passes the pointer event to it's handler, so the partner would become this object instead of just undefined, which was causing other problems. I reverted the change